src/Apps/Marketing/Routes/MarketingMeetArtAdvisorRoute.tsx

```diff
@@ -78,26 +81,28 @@ export const MarketingMeetArtAdvisorRoute: FC = () => {
           ]}
         />
 
-        <FullBleed bg={["black5", "transparent"]} py={[4, 0]} px={[2, 4]}>
-          <GridColumns gridRowGap={4}>
-            <Column span={12} textAlign="center">
-              <Text variant="xl">Meet your new art advisor. It’s Artsy.</Text>
-            </Column>
+        {!!isEigen && (
```

Comment:
We want it to show up if the client is not Eigen, right? 👀 
```suggestion
        {!isEigen && (
```

Reply:
Exactly! I've committed your fix.